lib/twitch.rb

```diff
@@ -41,21 +41,34 @@ def get(id, cursor)
       end
 
       def route(id, cursor)
-        User.route(id)["/follows/channels?limit=100&cursor=#{cursor}"]
+        Twitch.route["/users/follows?from_id=#{id}&after=#{cursor}"]
+      end
+    end
+  end
+
+  module Videos
+    def self.recent(id, limit: 10)
+      JSON.parse(get(id))['data']
+    end
+
+    class << self
+      def get(id)
+        route(id).get(Twitch.headers)
+      end
+
+      def route(id)
+        Twitch.route["/videos?user_id=#{id}"]
```

Comment:
Only archive videos should be searched. The timestamp on highlights indicates when the highlight was saved, not when it was recorded.

Reply:
Ohhh dang, that would explain why I thought it was 90 minutes off. I guess that's how long it takes me to highlight my PBs after the stream :O 

Until now I was actually planning on changing this to search only highlights, because even if an archive is found it will expire eventually. So it sounds like this feature is mostly useless as-is, since archives aren't good enough and highlights can't be found accurately. So I think I should add the highlight link to this branch.